src/plugins/intel_gpu/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -9269,9 +9309,9 @@ TEST(convolution_gpu_onednn, quantized_onednn_convolution_u8s8f32_asymmetric_act
         return;
 
     auto input = engine.allocate_memory({ data_types::u8, format::bfyx, { 1, 2, 5, 4 } });
-    auto weights = engine.allocate_memory({ data_types::i8, format::bfyx, { 2, 2, 3, 3 } });
-    auto biases = engine.allocate_memory({ data_types::f32, format::bfyx, { 1, 2, 1, 1 } });
-    auto a_zp = engine.allocate_memory({ data_types::u8, format::bfyx, { 1, 2, 1, 1 } });
+    auto weights = engine.allocate_memory({ data_types::i8, format::bfyx, { 3, 2, 3, 3 } });
```

Comment:
NIT: As I remember, you change shape 2 to 3, because of onednn issue. Add comment for the reason will helpful for colleagues looking at the PR or the code later.

Reply:
I will add relevant msg.